packages/cdktf/test/cached-assets.test.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) HashiCorp, Inc
+// SPDX-License-Identifier: MPL-2.0
+import { TerraformStack, Testing } from "../lib";
+import * as path from "path";
+import { TerraformCachedAsset } from "../lib/terraform-cached-asset";
+import * as fs from "fs";
+
+describe("cached asset", () => {
+  afterEach(() => {
+    try {
+      fs.rmSync(path.join(__dirname, "fixtures", "cached-asset", "output"), {
+        recursive: true,
```

Comment:
By passing `force: true` ([docs](https://nodejs.org/api/fs.html#fsrmsyncpath-options)) we can get rid of the try-catch here – this would help us to debug this test case if something is weird with the filesystem (else the error will just get swallowed).

Reply:
👍